### PR TITLE
Add newsletter to Python section

### PR DESCRIPTION
**Pull request by khuyentran1401**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,6 +253,7 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 - [Weekly Python Newsletter](https://importpython.com/newsletter/). Weekly Python Newsletter containing Python Articles, Projects, Videos, Tweets delivered in your inbox. [Archive](https://importpython.com/newsletter/archive/).
 - [Pycoders Weekly](https://pycoders.com/). A free weekly e-mail newsletter, on Fridays, for those interested in python development and various topics around python.
 - [Awesome Python Newsletter](https://python.libhunt.com/newsletter). A weekly overview of the most popular Python news, articles and packages.
+- [Data Science Simplified](https://mathdatasimplified.com/). A daily Python and data science snippet
```
Comment (ferhatelmas) [inline README.md:256]: could you fix the format? `.` at the end.

Comment (khuyentran1401) [reply README.md]: I have just added it. Let me know if it looks good to you

**Review by ferhatelmas — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (ferhatelmas) [reply README.md]: sure, thanks

**Pull request merged**